src/verify.c

```diff
@@ -606,6 +606,8 @@ int verify_main(int argc, char **argv)
 	struct manifest *official_manifest = NULL;
 	int ret;
 	int lock_fd;
+	int retries = 0;
+	int timeout = 5;
```

Comment:
Okay. I was asking because you're adding "10" and "5" in the same commit but don't explain why.